tron/api/resource.py

```diff
@@ -406,25 +406,25 @@ def __init__(self, mcp):
 
         # Setup children
         self.putChild(
-            'jobs',
+            b'jobs',
```

Comment:
Do we really need this? We already have unicode_literals everywhere, so I would have expected that the current version wouldn't work if we needed bytes here.

Reply:
yeah, i was surprised, but apparently everything URL-related is still handled as bytes in twisted